gorapass/urls.py

```diff
@@ -6,6 +6,7 @@
     path('', views.index, name='index'),
     path('hikes', views.hikes, name='hikes'),
     path('stamps', views.stamps, name='stamps'),
+    path('stamps/<int:stamp_id>', views.stamp, name='stamps'),
```

Comment:
This has the same name as the path above it. We should probably use just 'stamp' for this one instead of 'stamps'

Reply:
Good catch